Add hint on removing SUSE products (#2297)
theforeman/foreman-documentation: guides/common/modules/proc_installing-the-scc-manager.adoc

[id="Installing_the_SCC_Manager_Plugin_{context}"]
= Installing the SCC Manager Plug-in

> You can use the following procedure to install the _SCC Manager_ plug-in on your {Project}.

Reviewer: ```suggestion
Use this procedure to install the _SCC Manager_ plug-in on your {Project}.
```

Author: Applied.
